ksp_plugin/vessel.hpp

```diff
@@ -163,6 +163,14 @@ class Vessel {
       not_null<Ephemeris<Barycentric>*> ephemeris,
       not_null<Celestial const*> parent);
 
+  void AppendToPsychohistory(
+      Instant const& time,
+      DegreesOfFreedom<Barycentric> const& degrees_of_freedom,
+      bool authoritative);
+
+  DiscreteTrajectory<Barycentric> const& psychohistory() const;
+  bool psychohistory_is_history() const;
```

Comment:
This is a rather confusing identifier.  I would prefer `psychohistory_is_authoritative`.